includes/views/graceful-degradation.php

```diff
@@ -0,0 +1,3 @@
+<div class="notice notice-error is-dismissible">
+    <p><?php _e( 'Repeatable content mapping requires the installation of Advanced Custom Fields Pro WordPress Plugin. To enable this functionality, please install and refresh the page.', 'your-text-domain' ); ?></p>
```

Comment:
What should `your-text-domain` be?

Reply:
@pat-kavanagh I think `gathercontent-import` would be the most consistent text-domain to use here.